Moved Cairo code snippets to Scarb projects in `docs/listings` (#2470)
foundry-rs/starknet-foundry: docs/src/snforge-advanced-features/conditional-compilation.md

# Conditional Compilation

> 📝 **Note**

> > For an elaborative guide on Scarb conditional compilation, please refer to [Scarb documentation](https://docs.swmansion.com/scarb/docs/reference/conditional-compilation.html)

Reviewer: ```suggestion
> For more detailed guide on Scarb conditional compilation, please refer to [Scarb documentation](https://docs.swmansion.com/scarb/docs/reference/conditional-compilation.html)
```
